### ISSUE TITLE
Curator - Karazhan

### ISSUE THREAD

**Issue by xfurry** [scriptdev2/scriptdev2#145]
Note to myself (or someone who is willing to fix this).

Npc http://www.wowhead.com/npc=15691/the-curator is interrupting spell evocation cast https://github.com/scriptdev2/scriptdev2/blob/master/scripts/eastern_kingdoms/karazhan/boss_curator.cpp#L166 after cmangos https://github.com/cmangos/mangos-wotlk/commit/5330f918b26c5e1a54eb56a3f686b66bb7d51c7a

**Comment by ghost** [comment]
Also Curator spamm Berserk text and spell. https://github.com/Ulduar/scriptdev2-cata/commit/b23265032255bbf6a483f9ba27c46afcfeb1727e#diff-1e29381a41dba3f52ea0e84875802a6eR134

**Comment by xfurry** [comment]
@Ulduar one thing which I don't like on your commits, is that you are adding a lot of random stuff into one single commit. This makes things very hard to follow...
I will check the Curator problem soon, as it's most likely a core issue.
